client/src/components/ProfileEdit.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import { Row, Col, Input, Button } from 'react-materialize'
+import { request } from '../utils'
+import { Link } from 'react-router-dom'
+
+
+class ProfileEdit extends Component {
+
+	state = {
+		  user_edit: {},
+      error_nick: "",
+      error_repeated: ""
+	}
+
+	componentDidMount() {
+		request('/profile/' + this.props.match.params.profile_id, "GET")
+    .then(json => {
+			this.setState({user_edit: json});
+      console.log(this.state.user_edit)
+      this.setState({
+        user_edit: {
+          f_name: this.state.user_edit.first_name,
+          l_name: this.state.user_edit.last_name,
+          nick: this.state.user_edit.nickname,
+          loc: this.state.user_edit.location,
+          pass: this.state.user_edit.password
+        }
+      })
+		})
+	}
+
+	UpdateClick = (eve) => {
+		eve.preventDefault()
+
+    let nick_error = ""
+    let repeated_error = ""
```

Comment:
Please do not use tabs, and replace them by spaces. It makes hard to indent lines properly.